frontend/rust-lib/flowy-document2/src/event_map.rs

```diff
@@ -76,4 +78,7 @@ pub enum DocumentEvent {
 
   #[event(input = "TextDeltaPayloadPB")]
   ApplyTextDeltaEvent = 11,
+
+  #[event(input = "CopyPayloadPB", output = "CopyResponsePB")]
+  Copy = 12,
```

Comment:
document what it does

Reply:
There are no comments for other events here, the comments are all in the handler